testsuite/features/support/constants.rb

```diff
@@ -48,6 +48,8 @@
                     'rocky8_ssh_minion' => 'ROCKY8_SSHMINION',
                     'rocky9_minion' => 'ROCKY9_MINION',
                     'rocky9_ssh_minion' => 'ROCKY9_SSHMINION',
+                    'ubuntu1804_minion' => 'UBUNTU1804_MINION',
```

Comment:
Why Ubuntu 18.04, which is in EoL already was appended here?

Reply:
This was not intentional. I guess this was lost in my fight with git merge